### DISCUSSION TITLE
"Export using previous settings" not exporting anything...

### DISCUSSION THREAD

**Discussion by cgi-joe** [KosmosisDire/obsidian-webpage-export#542] [General]
Has anyone else run into this issue?

I am trying to use a hotkey to export my notes (ideally just the changed ones), but when I use this feature, it seems to process the export but skip over any changed files. Has anyone else experienced this issue? For what it's worth I have tried both the official release and the BRAT beta version as well. Any hope would be appreciated - this is the last missing piece of the puzzle that will allow me to fully automate my obsidian to neocities pipeline - thanks in advance!

**Comment by KosmosisDire** [comment]
Hi, if you execute the export using the export modal (by pressing the export button in the ribbon) and then click export there does it work there and are all the files you expect selected? Is it only breaking when using the command specifically?
